pkg/scheduler/generic_scheduler.go

```diff
@@ -407,9 +407,9 @@ func prioritizeNodes(
 	// This is required to generate the priority list in the required format
 	if len(extenders) == 0 && !fwk.HasScorePlugins() {
 		result := make(framework.NodeScoreList, 0, len(nodes))
-		for i := range nodes {
+		for _, node := range nodes {
 			result = append(result, framework.NodeScore{
-				Name:  nodes[i].Name,
+				Name:  node.Name,
```

Comment:
looks like the different way to iterate a slice but I cannot see where is the optimization comes from.

Reply:
It’s just that I think it’s cleaner to iterate slice the code like this : ) If not, I will close this pr.🙈